Microsoft.Recognizers.Text.NumberWithUnit.Tests/TestNumberWithUnitChinese.cs

```diff
@@ -143,24 +143,9 @@ public void TestAge()
                       "这件事发生在宝宝只有十个月大的时候.",
                       "10 Month");
 
-            /*
-            BasicTest(model,
-                      "如果我们在十二月份开始，会有三周时间。",
-                      "3 Week");
-
-            BasicTest(model,
-                      "这封邮件已经发送出去15天了，回信在哪里?",
-                      "15 Day");
```

Comment:
If the sentences in Chinese don't make sense. Please add new ones for similar cases to be tested.